legacy/f4mp_server/f4mp_server.vcxproj

```
@@ -41,7 +41,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v142</PlatformToolset>
+    <PlatformToolset>v140</PlatformToolset>
```

`v142` not compatible with VS2015
```suggestion
    <PlatformToolset>v140</PlatformToolset>
```